lighthouse-core/report/html/report-styles.css

```diff
@@ -406,22 +407,44 @@
 .lh-load-opportunity {
   padding-top: var(--lh-audit-vpadding);
   padding-bottom: var(--lh-audit-vpadding);
-  border-top: 1px solid var(--report-secondary-border-color);
+  border-bottom: 1px solid var(--report-secondary-border-color);
 }
 
 .lh-load-opportunity:last-of-type {
   border-bottom: none;
 }
 
-.lh-load-opportunity__summary {
+.lh-load-opportunity__summary,
+.lh-load-opportunity__header {
   display: flex;
   align-items: flex-start;
-  flex-wrap: wrap;
-  min-height: calc(var(--body-line-height) + var(--caption-line-height));
 }
 
-.lh-load-opportunity__summary .lh-toggle-arrow {
-  margin-top: calc((var(--subheader-line-height) - 12px) / 2);
+.lh-load-opportunity__header .lh-load-opportunity__col {
+  background-color: var(--medium-50-gray);
+  color: var(--medium-75-gray);
+  text-align: center;
+  display: unset;
+  line-height: 40px;
+}
+
+.lh-load-opportunity__summary {
+  padding-right: var(--text-indent);
+}
+
+.lh-load-opportunity__col {
+  display: flex;
+  justify-content: space-between;
+}
+.lh-load-opportunity__col > * {
+  margin: 0 5px;
+}
+.lh-load-opportunity__col--one {
+  flex: 5;
```

Comment:
I must say this was slightly magical to me given the header and details have different content, I have much to learn 📖 🤓